gitbook-docs/README.md

```diff
@@ -47,23 +53,29 @@ Signal K defines methods for combining arbitrary data from the Data Model into v
 UTF-8 JSON format.
 
 Rather than define hundreds of specific messages, Signal K has a few common message formats which can contain any
-combination of data from the Data Model. This means that we don’t need new messages for every new case, just extra data
-in the payload. It means that any device can read any message and a device can introduce a new data point which can be
+combination of data from the Data Model. This means that there is no need to define new messages for every new data
+point or command which needs to be supported, rather the message format remains the same and only the data transmitted
+needs to change. It means that any device can read any message and a device can introduce a new data point which can be
 understood by existing devices without the need for firmware upgrades.
 
 ## Signal K Transport Layer
 
-Signal K does not define the transport or wire protocol. Signal K messages are JSON text and can be sent over almost any
-transport layer. However, we do provide guidance on how to make an initial connection, handle negotiation,
-subscription, and disconnection for a given transport (e.g. TCP/IP or serial).
+Signal K does not define the transport or wire protocol. Signal K messages are JSON text and can be sent over almost
+any physical transport layer. However, the Signal K standard does provide guidance on how to establish an initial
+connection, handle negotiation, subscription, and disconnection for a given transport (e.g. TCP/IP or serial).
 
-Where possible we use well established standards like HTTPS, REST, WebSockets, MQTT, STOMP, etc. But within each method
-there are always many options in message addressing, formating, or transfer (GET, POST), etc.
+Where possible Signal K uses well established standards like HTTPS, REST, WebSockets, MQTT, STOMP, etc. However, these
```

Comment:
This implies that we have defined MQTT and STOMP mappings, which is not the case. I'd remove them here.